catroid/src/test/java/org/catrobat/catroid/test/merge/MergeTaskTest.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2018 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.catrobat.catroid.test.merge;
+
+import android.content.Context;
+
+import org.catrobat.catroid.common.FlavoredConstants;
+import org.catrobat.catroid.content.Project;
+import org.catrobat.catroid.content.Scene;
+import org.catrobat.catroid.formulaeditor.UserList;
+import org.catrobat.catroid.formulaeditor.UserVariable;
+import org.catrobat.catroid.io.StorageOperations;
+import org.catrobat.catroid.merge.MergeTask;
+import org.catrobat.catroid.test.MockUtil;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+
+import static junit.framework.Assert.assertFalse;
+import static junit.framework.TestCase.assertTrue;
+
+public class MergeTaskTest {
+	private Project firstProject;
+	private Project secondProject;
+	private MergeTask mergeTask;
+
+	@Before
+	public void setUp() {
+		Context mockContext = MockUtil.mockContextForProject();
+		firstProject = new Project(mockContext, "testProjectOne");
+		secondProject = new Project(mockContext, "testProjectTwo");
+		firstProject.addUserList(new UserList("userListOne"));
+		firstProject.addUserVariable(new UserVariable("userVariableOne"));
+		firstProject.addScene(new Scene("sceneOne", firstProject));
+		secondProject.addUserList(new UserList("userListTwo"));
+		secondProject.addUserVariable(new UserVariable("userVariableTwo"));
+		secondProject.addScene(new Scene("sceneTwo", secondProject));
+		mergeTask = new MergeTask(firstProject, secondProject, mockContext, false);
+	}
+
+	@After
+	public void tearDown() throws IOException {
+		deleteProject(firstProject.getName());
+		deleteProject(secondProject.getName());
+		deleteProject("mergedProject");
+	}
+
+	private void deleteProject(String projectName) throws IOException {
+		File projectDir = new File(FlavoredConstants.DEFAULT_ROOT_DIRECTORY, projectName);
```

Comment:
Same problem as deleteProject in ConflictHelperTest.java 

You can try out for example in `tearDown()` comment out all three `deleteProject(...)` functions and you actually will pass 3 of 4 tests.